OnebyAOL/MillennialNativeAdAdapter.m

```
@@ -45,11 +45,13 @@ - (instancetype)initWithMMNativeAd:(MMNativeAd *)ad {
         }
 
         if (ad.mainImageView.image) {
```

We should change the logic to fail if any of the assets are not present. Otherwise we will run into the same issue of serving failed image native ads

---

This is resolved.